package.json

```diff
@@ -215,10 +215,10 @@
     ],
     "coverageThreshold": {
       "global": {
-        "statements": 84,
-        "branches": 83,
-        "functions": 80,
-        "lines": 84
+        "statements": 92.82,
```

Comment:
Why these numbers? And why so specific? Instead of `92.82` not `93`?

Reply:
als je dan een stukje code toevoegt zonder test dan faalt de build meteen. dat is wat je juist wil volgens mij.